src/flows/components/panel/FlowPanel.tsx

```diff
@@ -38,6 +45,12 @@ export interface Props extends PipeContextProps {
 export const DEFAULT_RESIZER_HEIGHT = 360
 export const MINIMUM_RESIZER_HEIGHT = 220
 
+const PANEL_PREVIEW_TYPES = new Set([
```

Comment:
This is counter to the self registration patterns in notebooks. Can we key off of family instead? Or use a pattern like we have for resizes?